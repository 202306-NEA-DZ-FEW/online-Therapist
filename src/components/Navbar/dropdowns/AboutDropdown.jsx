import Link from "next/link";
import React from "react";
import { useTranslation } from "next-i18next";

export default function AboutDropdown() {
    const { t } = useTranslation("common");
    return (
        <div className='md:w-36 w-48 bg-white font-atkinson border p-2 shadow-md rounded-lg '>
            <ul className='py-2 first:pt-0 z-60 last:pb-0'>
                <Link href='/about-us/'>
                    <li
                        className=' py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700'
                        href='#'
                    >
                        {t("navbar.aboutUs")}
                    </li>
                </Link>
                <Link href='/team/'>
                    <li
                        className=' py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700'
                        href='#'
                    >
                        {t("navbar.team")}
                    </li>
                </Link>
                <Link href='/careers/'>
                    <li
                        className=' py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700'
                        href='#'
                    >
                        {t("navbar.careers")}
                    </li>
                </Link>
            </ul>
        </div>
    );
}
