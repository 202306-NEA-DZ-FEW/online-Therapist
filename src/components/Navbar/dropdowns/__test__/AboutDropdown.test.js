import renderer from "react-test-renderer";
import AboutDropdown from "../AboutDropdown";

it("renders correctly", () => {
    const tree = renderer.create(<AboutDropdown />).toJSON();
    expect(tree).toMatchSnapshot();
});
