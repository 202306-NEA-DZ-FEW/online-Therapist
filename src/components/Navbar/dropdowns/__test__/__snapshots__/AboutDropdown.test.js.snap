// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="md:w-36 w-96 bg-white font-atkinson border p-2 shadow-md rounded-lg "
>
  <ul
    className="py-2 first:pt-0 z-60 last:pb-0"
  >
    <a
      href="/about-us"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      <li
        className=" py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700"
        href="#"
      >
        navbar.aboutUs
      </li>
    </a>
    <a
      href="/team"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      <li
        className=" py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700"
        href="#"
      >
        navbar.team
      </li>
    </a>
    <a
      href="/careers"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      <li
        className=" py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700"
        href="#"
      >
        navbar.careers
      </li>
    </a>
  </ul>
</div>
`;
